proposals/description/verify_rari_proposal.ts

```diff
@@ -0,0 +1,96 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const fip_x: TemplatedProposalDescription = {
+  title: 'Verify Rari deprecation proposal',
+  commands: [
+    {
+      target: 'rariUSDCPoolControllerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolControllerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariRSPTFundManagerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolManagerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
```

Comment:
what do each of these actions do

Reply:
have to setFundRebalancer to the timelock address so the timelock can then claim the fuse fees